.github/workflows/ibm-nightly-warp-vm-cleanup.yaml

```
@@ -0,0 +1,95 @@
+            echo "🗑️ Releasing floating IP(s)"
+            for IP_ID in ${FLOATING_IP_IDS}
+            do
+              ibmcloud is floating-ip-release "${IP_ID}" --force >/dev/null
+            done
+            echo "✅ All tagged floating IPs released successfully"
+          else
+            echo "ℹ️  No floating IPs found"
+          fi
+
+      - name: Clean up IBM VMs
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:25:5: Double quote to prevent globbing and word splitting

(shellcheck)